.github/workflows/unittests.yml

```diff
@@ -29,6 +30,10 @@ jobs:
 
       - uses: jasp-stats/jasp-actions/setup-test-env@master
 
+      - name: Make sure igraph is installed from source
+        run: install.packages("igraph", type = "source", repos = "https://cloud.r-project.org/")
```

Comment:
well for some reason the igraph dll/ .so from the rstudio package binary doesn't work on ubuntu. That's why I install it from source. We don't use this repository for flatpak, so I don't think it matters for JASP. I'm only using this repository to speed up package installation btw.

Reply:
Can we add comments to these yaml files? If we can, it'd be good to add the reasoning, because I can totally see us forgetting why we have this in the future

Reply:
sure, I'll add a comment about it. I do think it's somewhat self-explanatory though, if you remove it you'll just notice that the test crashes on ubuntu.